schemas/layer-3.cue

```diff
@@ -53,59 +70,81 @@ package schemas
 	providers?: [...string]
 }
 
+// Layer 3 specific mapping that extends common Mapping with modifications.
+// Used by ImportedPolicies, GuidanceReferences, and ControlReferences.
+// Modifications chain sequentially when importing policies (e.g., Base → Parent → Child), preserving all modifications.
 #PolicyMapping: {
-	"reference-id": string @go(ReferenceId) @yaml("reference-id",omitempty)
-	"in-scope":     #Scope @go(InScope) @yaml("in-scope",omitempty)
```

Comment:
Actually rethinking this deletion. We might need the scope of the mapping if we need to customize per technology, provider, or boundaries as a subset of the global scope.